exts/cesium.usd.plugins/cesium/usd/plugins/CesiumUsdSchemas/__init__.pyi

```diff
@@ -6,19 +6,15 @@ import pxr.UsdGeom
 
 __MFB_FULL_PACKAGE_NAME: str
 
-class CartographicPolygon(pxr.UsdGeom.Gprim):
+class CartographicPolygon(pxr.UsdGeom.BasisCurves):
```

Comment:
This should already be in the main branch. Are the `__init__.py` files manually maintained?

I just checked out main and ran `regenerate_schema.bat` followed by `genStubs.sh`. This file wasn't created, but I have errors in `genStubs.sh`

Reply:
These changes came from me running `genStubs.bat`, no manual edits. Are you able to run the `.bat` version?

Reply:
Yes, but in the past only partially successfully. I would get errors, but after some fixing with @weegeekps the errors didn't block compilation. Maybe the errors prevented some generated changes from happening. The error was consistent, but as of today I'm not able to reproduce them. Maybe there was a patch. I'll keep an eye on it next time I edit the schemas.